sumOfOther/test.js

```diff
@@ -0,0 +1,25 @@
+const assert = require('assert');
+const sumOfOther = require('./sumOfOther.js');
+
+describe('sumOfOther', () => {
+  it('1', () => {
```

Comment:
This function is pretty strait forward, therefore it is difficult to create some edge case tests.

But again, you could combine all `it` blocks into one and name it properly. Also you could add test for empty array, or for undefined and etc.